src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,23 @@
+.stars {
+  margin: 0;
+  padding: 0;
+  list-style: none;
+  display: flex;
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    background-position: center;
+    background-repeat: no-repeat;
+    background-image: url('../images/star.svg');
+    margin-right: 4px;
+  }
+
+  &__star:last-child {
+    margin-right: 0;
+  }
+}
+
+.stars--4 .stars__star:nth-child(-n + 4) {
+  background-image: url('../images/star-active.svg');
+}
```

Comment:
You should reuse logic from `stars` task. So If I change modifier to `stars--2` I should see only 2 active stars